registry/donor/views.py

```diff
@@ -172,3 +185,52 @@ def save_note():
     db.session.commit()
     flash("Poznámka uložena.", "success")
     return redirect(url_for("donor.detail", rc=note_form.rodne_cislo.data))
+
+
+@blueprint.get("/donor/ignore")
+@login_required
+def show_ignored():
+    ignored = IgnoredDonors.query.all()
+    return render_template(
+        "donor/ignore_donor.html",
+        ignore_form=IgnoreDonorForm(),
+        ignored=ignored,
+        unignore_form=RemoveFromIgnoredForm(),
+    )
+
+
+@blueprint.post("/donor/ignore/add")
+@login_required
+def ignore_donor():
+    ignore_form = IgnoreDonorForm()
+    if ignore_form.validate_on_submit():
+        if not IgnoredDonors.query.get(ignore_form.rodne_cislo.data):
+            ignored = IgnoredDonors(
+                rodne_cislo=ignore_form.rodne_cislo.data,
+                reason=ignore_form.reason.data,
+                ignored_since=datetime.now(),
+            )
+            db.session.add(ignored)
+            db.session.commit()
+            DonorsOverview.remove_ignored()
+            flash("Dárce ignorován.", "success")
+        else:
+            flash("Dárce již je v seznamu ignorovaných", "danger")
+    else:
+        flash("Při přidávání do ignorovaných došlo k chybě", "danger")
+    return redirect(url_for("donor.show_ignored"))
+
+
+@blueprint.post("/donor/ignore/remove")
+@login_required
+def unignore_donor():
+    unignore_form = RemoveFromIgnoredForm()
+    if unignore_form.validate_on_submit():
+        ignored_donor = IgnoredDonors.query.get(unignore_form.rodne_cislo.data)
+        db.session.delete(ignored_donor)
```

Comment:
Teoreticky by mělo jít zkrátit na:
```suggestion
        IgnoredDonors.query.get(unignore_form.rodne_cislo.data).delete()
```

Reply:
Nejde
`AttributeError: 'IgnoredDonors' object has no attribute 'delete'`

Reply:
Asi by to fungovalo s .filter, ale to by zase bylo zbytečně dlouhé, ok, nechme tak jak to je.